hw03_CollegeSteven.py

```diff
@@ -0,0 +1,67 @@
+#################################################################################
+# Author: Steven Lintelman-Nader
+# Username: CollegeSteven
+#
+# Assignment: Homework 3 - Main
+# Purpose: Better understanding Git as a collaborative space.
+# Google Doc Link: https://docs.google.com/document/d/15SLmGKKaDWuRTPArJhV6hO5h7I_5wTs5DAcg8XJPokU/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+import turtle
+
+
+def cake(x, y, height, length):
+    """
+    Given a position near the top left of the cake, height and length, creates a cake or pink fortress with a turtle
+    """
+
+    # Create the initial top icing decoration.
+    c = turtle.Turtle()
+    c.hideturtle()
+    c.teleport(x, y)
```

Comment:
I do have to get into the habit of making acknowledgements. I just pushed those changes. I hope what I added works, if not I can make another adjustment of course. I added the link to python turtle graphics documentation, just in case.